common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -94,7 +95,32 @@ def save_subs_to_store(subs, subs_id, item, language='en'):
     return save_to_store(filedata, filename, 'application/json', item.location)
 
 
-def get_transcripts_from_youtube(youtube_id, settings, i18n):
+def transcript_name_youtube_video(youtube_text_api):
+    """
+    Get the transcript name from available transcripts of video
+    with respect to language from youtube server
+    """
+    transcripts_param = {'type': 'list', 'v': youtube_text_api['params']['v']}
+    lang = youtube_text_api['params']['lang']
+    # get list of transcripts of specific video
+    # url-form
+    # http://video.google.com/timedtext?type=list&v={VideoId}
+    youtube_response = requests.get('http://' + youtube_text_api['url'], params=transcripts_param)
+    transcript_name = None
+    if youtube_response.status_code == 200 and youtube_response.text:
+        # convert to order-dict youtube server response
+        youtube_data = xmltodict.parse(youtube_response.text)
```

Comment:
elsewhere, we parse the `youtube_response.text` with `lxml.etree`. Please use `etree` instead of `xmltodict` for consistency.